### PR TITLE
MRG: create CONTRIBUTING.rst

### PR DESCRIPTION
**Pull request by jJasonWang**
Created CONTRIBUTING.rst similar to [permute](https://github.com/statlab/permute/blob/master/CONTRIBUTE.rst).

@aksam-ahmad, please review.

### CODE REVIEW

**Comment by jarrodmillman** [comment]
I've merged a few other pull requests since you branched off master.  I will merge this once you've rebased this off master.  You will want to do something like this:

$ git checkout master
$ git pull <upstream> master  # where <upstream> is the name you gave to the github.com/berkeley-stat222/mousestyles
$ git checkout <this branches name>
$ git rebase master

Then you will need to force (since you rewrote history) your push to <the jJasonWang remote> on <this branches name>.

**Comment by jJasonWang** [comment]
I rebased those commits. Thanks for reminding me. I will let @aksam-ahmad to change the title to `MRG` once he finished the review.

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+
+Working with ``np.random``
+--------------------------
+
+If you need to set a random seed, do **not** use ``np.random.seed()``.  Using
+``np.random.seed()`` can produce inconsistent results.  Instead you should use
+:func:`get_prng` with your chosen seed.  Here is a tiny
+example of how you might create a function that generates random numbers::
+
+    from mousestyles.utils import get_prng
```
Comment (aksam-ahmad) [inline CONTRIBUTING.rst]: The 'get_prng' function is associated with the **permute** package. We can consider removing this final section after discussing with @jarrodmillman.

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+To measure the test coverage, install
+`coverage.py <http://nedbatchelder.com/code/coverage/>`__
+(e.g., using ``pip install coverage``) and then run::
+
+  $ make coverage
+
+This will print a report with one line for each file in `mousestyles`,
+detailing the test coverage::
+
+  Name                 Stmts   Miss Branch BrMiss  Cover   Missing
```
Comment (aksam-ahmad) [inline CONTRIBUTING.rst]: These tests are from *permute*. We can either remove or add the mousestyles output.

**Pull request merged**